README.md

```diff
@@ -40,50 +56,26 @@ Run type [automatic/manual]: manual
-```
+docker-locust can be run in multiple docker-containers. It is useful for users who has more than one machine to create bigger load. In this point we are using docker-compose, but it is also possible to run it in different ways, e.g. Cloudformation in AWS.
 
-Sample command to read load test script from local machine:
+Run the application with the command:
 
 ```bash
-bash <(curl -s https://raw.githubusercontent.com/zalando-incubator/docker-locust/master/local.sh) deploy https://targeturl.com myfolder/myscript.py 4 manual
+bash <(curl -s https://raw.githubusercontent.com/zalando-incubator/docker-locust/master/local.sh) deploy
 ```
 
 Read multiple files
 -------------------
 docker-locust has the ability to read multiple files from s3 or any http/https, e.g. [1 file is the load test file / python file](https://raw.githubusercontent.com/zalando-incubator/docker-locust/master/example/simple_post.py) and [1 other file is json file](https://raw.githubusercontent.com/zalando-incubator/docker-locust/master/example/payloads.json) where payloads are stored. Sample command:
 
 ```bash
-bash <(curl -s https://raw.githubusercontent.com/zalando-incubator/docker-locust/master/local.sh) deploy https://targeturl.com https://raw.githubusercontent.com/zalando-incubator/docker-locust/master/example/simple_post.py,https://raw.githubusercontent.com/zalando-incubator/docker-locust/master/example/payloads.json 4 manual
-```
-
-Run as standalone version
--------------------------
-
-docker-locust can be also run as **standalone version**, which required only 1 single docker container.
-
-Sample command to read load test script from any http/https url:
-
-```bash
-docker run -i -p 8089:8089 -e TARGET_HOST=https://targeturl.com -e LOCUST_FILE=https://raw.githubusercontent.com/zalando-incubator/docker-locust/master/example/simple.py -e ROLE=standalone -e AUTOMATIC=True -e DURATION=10 -e HATCH_RATE=1 -e USERS=5 registry.opensource.zalan.do/tip/docker-locust
-```
-
-Sample command to read load test script from S3 bucket:
-
-```bash
-docker run -i -v ~/.aws:/root/.aws -p 8089:8089 -e TARGET_HOST=https://targeturl.com -e LOCUST_FILE=s3://mybucket/mypath/myscript.py -e ROLE=standalone -e AUTOMATIC=True -e DURATION=10 -e HATCH_RATE=1 -e USERS=5 registry.opensource.zalan.do/tip/docker-locust
-```
-
-Sample command to read load test script from local machine:
-
-```bash
-docker run -i -v $PWD/example:/opt/script -p 8089:8089 -e TARGET_HOST=https://targeturl.com -e LOCUST_FILE=simple.py -e ROLE=standalone -e AUTOMATIC=True -e DURATION=10 -e HATCH_RATE=1 -e USERS=5 registry.opensource.zalan.do/tip/docker-locust
+STANDALONE=true bash <(curl -s https://raw.githubusercontent.com/zalando-incubator/docker-locust/master/local.sh) deploy https://targeturl.com https://raw.githubusercontent.com/zalando-incubator/docker-locust/master/example/simple_post.py,https://raw.githubusercontent.com/zalando-incubator/docker-locust/master/example/payloads.json 4 manual
```

Comment:
Why don't you provide docker run examples and local.sh examples with named arguments?

Reply:
sure, I can do it

Reply:
great, thanks

Reply:
FYI: https://github.bus.zalan.do/TIP/ui-testing-with-zalenium-in-cdp-example might give some formatting ideas